runtime/lua/vim/_watch.lua

```diff
@@ -2,15 +2,15 @@ local M = {}
 local uv = vim.uv
 
 ---@enum vim._watch.FileChangeType
-local FileChangeType = {
+M.FileChangeType = {
   Created = 1,
   Changed = 2,
   Deleted = 3,
+  [1] = 'Created',
+  [2] = 'Changed',
+  [3] = 'Deleted',
 }
 
---- Enumeration describing the types of events watchers will emit.
```

Comment:
I think we should still keep the description.

Reply:
I dunno, it seems pretty redundant since know it's an enum, we know it's about types of events and it's defined in `_watch.lua` so must be used by watchers.

Reply:
Right but it's part of the "public API" of this module (and yeah the module itself is private but still).

Reply:
Done.